src/Models/ShallowWaterModels/calculate_shallow_water_tendencies.jl

```diff
@@ -32,6 +32,8 @@ function calculate_tendencies!(model::ShallowWaterModel)
                                                model.grid,
                                                model.gravitational_acceleration,
                                                model.advection,
+                                               model.tracer_advection,
+                                               model.height_advection,
```

Comment:
It might be better to use `advection.tracers` and `advection.mass`, so that we don't have to refactor this model too when we change the advection scheme implementations for nonhydrostatic and hydrostatic models in #2454

Reply:
its ok if you don't want to, just trying to be lazy